terraform/testcases/otlp_mock/otconfig.tpl

```diff
@@ -0,0 +1,24 @@
+extensions:
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:55680
+
+
+processors:
+
+exporters:
+  logging:
+    loglevel: debug
+  awsemf:
+    region: '${region}'
+    no_verify_ssl: false
+    endpoint: "${mock_endpoint}"
+
+service:
+  pipelines:
+    metrics:
+      receivers: [otlp]
+      exporters: [logging, awsemf]
```

Comment:
can we use the same metric pipeline config from our default collector configuration files?

Reply:
yes we can